### PR TITLE
Add refungible to terminology

### PR DESCRIPTION
**Pull request by usetech-llc**

### CODE REVIEW

**Review by okwme — DISMISSED** [review]

**Comment by okwme** [comment]
I don't have write access and can't resolve conflicts @ig-shaun

**Comment by okwme** [comment]
bump

File: NFT-REC-002-use-cases.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ The use case model and specific examples described here are not intended to be e
 | Metadata | Machine understandable information about web resources or other things |
 | Non-Fungible | Things that are uniquely identifiable individually, or as part of a set of things which are not equivalent, which makes them not interchangeable |
 | Referent | The [resource](https://www.w3.org/TR/rdf11-concepts/#dfn-resource) [denoted](https://www.w3.org/TR/rdf11-concepts/#dfn-denote) by an [IRI](https://www.w3.org/TR/rdf11-concepts/#dfn-iri) |
+| Re-fungible | Unique divisible things with shared ownership |
```
Comment (ig-shaun) [inline NFT-REC-002-use-cases.md]: @gregzaitsev would you agree to describe "things" as "resources", for consistency of the ontology?
From a legal perspective (not that I am a legal expert), the term "collective ownership" might be more precise confer the rights of collective ownership of a divisible asset, rather than "shared ownership", which I believe has different meaning.
Within the metadata sub-group we are currently discussing the semantics of ownership vs control of digital assets and resources, with a strong argument in favour of the capabilities of blockchains providing control vs ownership.

So I would propose the definition: "Unique resource, over which there is collective control by multiple parties"

Would be glad to get your further thoughts on this.

**Review by ig-shaun — CHANGES_REQUESTED** [review]
Proposed change to:

> Unique divisible resources, over which there is collective control by multiple parties.

File: NFT-REC-002-use-cases.md (hunk 1 above)
Comment (gregzaitsev) [reply NFT-REC-002-use-cases.md]: Yes, I agree! Done!

**Review by gregzaitsev — APPROVED** [review]

File: NFT-REC-002-use-cases.md (hunk 1 above)
Comment (lawyerphil) [reply NFT-REC-002-use-cases.md]: Disclaimer: This comment was prepared for general information purposes only. The information presented is not legal advice, is not to be acted on as such, may not be current and is subject to change without notice.

Shared Ownership legally refers to a non divisible property (eg. dwellings, automobiles, etc.) yet implies that one party is using the underlying asset ("lessee") and the other/s ("lessor") own/s the asset and allows the lessee to control the asset. The lessee has control and legal rights to the property (including legal liability). The lessor typically retains the right of entry (repossession). 

Collective Ownership does not have a standard legal implication. As such, there should still be some vetting on this, but it likely is the better of the two to use. Here the owners of the asset would each own the property in question for the same benefit of all members, rather than having owners possess differing control and ownership rights as 'shared ownership' does. This is how a court imo would define this term. 

TL;DR - 
Use collective ownership for members having equal ownership rights / control
Use shared ownership for members having differing ownership rights / control

Comment (ig-shaun) [reply NFT-REC-002-use-cases.md]: Thanks Phil, this is very helpful.
I'm in favour of using:

- Collective Ownership
- Shared Ownership

**Comment by ig-shaun** [comment]
The term Refungible has been replaced by more precise terminology
